services/QuillLMS/engines/evidence/lib/evidence/evidence/check/grammar.rb

```diff
@@ -4,8 +4,8 @@ module Evidence
   class Check::Grammar < Check::Base
 
     def run
-      grammar_client = Grammar::Client.new(entry: entry, prompt_text: prompt.text)
-      @response = Grammar::FeedbackAssembler.run(grammar_client.post)
+      grammar_client = ::Evidence::Grammar::Client.new(entry: entry, prompt_text: prompt.text)
```

Comment:
Note, this was giving me issues in my dev environment (the way classes are loaded), but isn't an issue in prod.